ui/src/pages/Group.svelte

```diff
@@ -70,6 +77,7 @@
       patp={host}
       key={group.keyObj}
       type="group"
+      isInstalledOrJoined={joinedDetails}
```

Comment:
when this is passed in, it's giving undefined, even if the group has been joined already. any ideas why?

Reply:
It's probably because it's only logging when the `Detail` component is initialised, not when the prop changes, try changing your log statement in `Detail.svelte` to be `$: console.log({isInstalledOrJoined})`

Reply:
okay that's right, the value does change from undefined to an object. but what's passed through to detail.svelte is still undefined, even after joinedDetails gets filled out. so the reactivity in svelte isn't working here.

shouldn't the detail component be re-rendered once `joinedDetails` changes, since `joinedDetails` is assigned inside a reactive statement with `loadGroup()` ?

Reply:
moved here https://github.com/worpet-bildet/portal/pull/155